src/main/java/com/streamr/client/dataunion/DataUnion.java

```diff
@@ -0,0 +1,233 @@
+package com.streamr.client.dataunion;
+
+import com.streamr.client.dataunion.contracts.DataUnionMainnet;
+import com.streamr.client.dataunion.contracts.DataUnionSidechain;
+import com.streamr.client.dataunion.contracts.IERC20;
+import com.streamr.client.utils.Web3jUtils;
+import org.apache.commons.lang3.reflect.FieldUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.web3j.abi.TypeEncoder;
+import org.web3j.abi.datatypes.*;
+import org.web3j.abi.datatypes.generated.Uint256;
+import org.web3j.crypto.Credentials;
+import org.web3j.crypto.ECKeyPair;
+import org.web3j.crypto.Sign;
+import org.web3j.protocol.Web3j;
+import org.web3j.protocol.core.methods.response.TransactionReceipt;
+import org.web3j.tx.Contract;
+import org.web3j.tx.ManagedTransaction;
+import org.web3j.utils.Numeric;
+
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.math.BigInteger;
+import java.util.ArrayList;
+
+import static com.streamr.client.utils.Web3jUtils.*;
+
+public class DataUnion {
+    private static final Logger log = LoggerFactory.getLogger(DataUnion.class);
+    public enum ActiveStatus{none, active, inactive};
```

Comment:
This enum describes the state of a member, could rename to MemberStatus. Items with Java enum naming convention: ACTIVE, INACTIVE, NONE.

Reply:
ok